feat: LEAP-1840: Add KeyPoints to COCO export

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/14924777069)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/451
<!-- Follow Merge -->

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14834452781

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14863228117

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-LEAP-1840'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14863228117

triklozoid commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14887727170)
> Successfully merged:  create mode 100644 web/libs/core/src/lib/utils/query-client.ts

triklozoid committed:
Merge branch 'develop' into 'fb-LEAP-1840'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14887727170

triklozoid committed:
Fix sdk commit

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14911550119

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-LEAP-1840'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14911550119

triklozoid committed:
Fix YOLO test

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-LEAP-1840'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14924777069

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14924777069
